ui-common/src/components/ServiceMapModal/ServiceMapModal.tsx

```diff
@@ -15,6 +15,7 @@ import { GraphData, ServiceMapGraph } from "./ServiceMapModalTypes"
 import { ResizableBox } from "react-resizable"
```

Comment:
change filteredServices to checkedServices
change filteredProtocols to checkedProtocols